src/layers/medCore/parameter/medStringParameter.cpp

```diff
@@ -35,15 +36,42 @@ QString medStringParameter::value() const
     return d->value;
 }
 
-void medStringParameter::setValue(const QString & value)
+void medStringParameter::setValue( QString const& value)
 {
     if(value != d->value)
     {
-        d->value = value;
-        emit valueChanged(d->value);
+       int i = -1;
+       if (d->poValidator == NULL || d->poValidator->validate(QString(value), i) == QValidator::Acceptable)
```

Comment:
Ne compile pas. Je suggere : 

       QString tmpValue = value;
       if (d->poValidator == NULL || d->poValidator->validate(tmpValue, i) == QValidator::Acceptable)

Reply:
Still to be corrected for compilation